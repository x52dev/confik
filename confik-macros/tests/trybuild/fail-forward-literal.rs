#[derive(confik::Configuration)]
#[confik(forward("hello world"))]
struct A;

fn main() {}
