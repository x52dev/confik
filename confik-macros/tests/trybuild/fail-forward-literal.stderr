error: expected identifier, found `"hello world"`
 --> tests/trybuild/fail-forward-literal.rs:2:18
  |
2 | #[confik(forward("hello world"))]
  |                  ^^^^^^^^^^^^^ expected identifier

error: proc-macro derive produced unparsable tokens
 --> tests/trybuild/fail-forward-literal.rs:1:10
  |
1 | #[derive(confik::Configuration)]
  |          ^^^^^^^^^^^^^^^^^^^^^

error[E0412]: cannot find type `AConfigBuilder` in this scope
 --> tests/trybuild/fail-forward-literal.rs:3:8
  |
3 | struct A;
  |        ^ not found in this scope
